packages/mobile/src/web3/saga.ts

```diff
@@ -48,30 +45,24 @@ let AssignAccountLock = false
 
 // checks if web3 claims it is currently syncing or not
 function* checkWeb3SyncProgressClaim() {
-  Logger.debug(TAG, 'Checking sync progress claim')
   while (true) {
     try {
-      const syncProgress = yield web3.eth.isSyncing()
-      Logger.debug(TAG, 'Sync progress', syncProgress)
-
+      const syncProgress = yield web3.eth.isSyncing() // returns true when it's still syncing and thus not ready
       if (typeof syncProgress === 'boolean' && !syncProgress) {
-        // For some weird reason, checkSyncProgressWorker is flaky and does not work for the long running
-        // sync tasks.
-        Logger.debug(TAG, 'sync complete')
+        Logger.debug(TAG, 'checkWeb3SyncProgressClaim', 'sync complete')
```

Comment:
Nit, we typically join the Tag and function name into one param (with a + sign or with string interpolation).
I guess this is more or less equivalent, not sure why we don't use this syntax more. Any thoughts?

Reply:
This is kinda inconsistent through the project, I like this one because it doesn't involve any extra code.